editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -53,15 +53,22 @@ static void _draw_shadowed_line(Control *p_control, const Point2 &p_from, const
 	p_control->draw_line(p_from + p_shadow_offset, p_from + p_size + p_shadow_offset, p_shadow_color);
 }
 
-void SpriteFramesEditor::_open_sprite_sheet() {
-	file_split_sheet->clear_filters();
-	List<String> extensions;
-	ResourceLoader::get_recognized_extensions_for_type("Texture2D", &extensions);
-	for (int i = 0; i < extensions.size(); i++) {
-		file_split_sheet->add_filter("*." + extensions[i]);
-	}
+void SpriteFramesEditor::_open_sprite_sheet(bool p_force_open) {
+	//If there's no texture registered then open the popup
```

Comment:
```suggestion
	// If there's no texture registered then open the popup.
```
[Comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide) nitpick